test/measurement_calibration/test_meas.py

```diff
@@ -186,7 +189,8 @@ def test_meas_cal_on_circuit(self):
         # Run the calibration circuits
         backend = Aer.get_backend('qasm_simulator')
         job = qiskit.execute(meas_calibs, backend=backend,
-                             shots=self.shots)
+                             shots=self.shots,
+                             seed_simulator=SEED)
```

Comment:
Does it make sense to use `seed_transpiler` here too?

Reply:
I am not sure if it will change much, but I guess that if it will change something - it better to have it more deterministic, so adding it can only help